syntax = "proto3";

service Subscriber {
    rpc Subscribe (SubscribeRequest) returns (SubscribeResponse);
}

message SubscribeRequest {
    string topic = 1;
    string host = 2;
    int32 port = 3;
}

message SubscribeResponse {
    bool is_success = 1;
}